Week1/homework/js-exercises/ArrayAnimals.js

```diff
@@ -0,0 +1,8 @@
+"use strict";
+var vehicles = [];
+console.log('These are some vehicles.');
+console.log(vehicles);
+var favAnimals = ['Cat', 'Dog', 'Elephant'];
+console.log(favAnimals);
+favAnimals.push('Piglet');
+console.log(favAnimals);
```

Comment:
Ah. Couple things here:
1. I am not sure if you are referring to any online material for JavaScript but it might be that it is a bit obsolete. This is because using var instead of let or const as we learnt in Week1 is the old way of doing things. Please try to use `let` or `const` instead of `var` now onwards.
2. 👍 for using `use strict`.